ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -111,25 +111,30 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-byo-route53
-- as: aws-ipi-efa-pg-f28
-  cron: 27 4 12 * *
+- as: aws-ipi-imdsv2-arm-f28
```

Comment:
this one is duplicated with the following `aws-ipi-imdsv2-fips-amd`, we just need one of them.

Reply:
removed aws-ipi-imdsv2-fips-amd

Reply:
Correct me, I removed aws-ipi-imdsv2-arm-f28